src/nimble_contact_entity.h

```diff
@@ -436,6 +446,8 @@ namespace nimble {
     //   next 3 bits are the face ordinal (range is 1-6)
     //   remaining bits are the genesis element id from the parent FEM mesh (e.g., the global id of the hex from which the face was extracted)
     int contact_entity_global_id_ = -1;
+
+    bool contact_status_ = false;
```

Comment:
Part of the changes in this PR add a contact_status_ member to ContactEntity which indicates the last iteration status. This is mostly used for visualization, but could be used for other purposes